### PR TITLE
Remove runOptions from docker profile

### PR DESCRIPTION
**Pull request by heuermh**
WIP, removing `runOptions` from the test docker profile to see what fails, if anything

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `<SOFTWARE>.version.txt` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Comment by heuermh** [comment]
Hmm, our CI config is too clever 😉 

Is there any way to force CI all the module tests?

File: .github/workflows/pytest-workflow-all.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+      - name: Upload logs on failure
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: logs-${{ matrix.profile }}-${{ matrix.nxf_version }}
+          path: |
+            /home/runner/pytest_workflow_*/*/.nextflow.log
+            /home/runner/pytest_workflow_*/*/log.out
+            /home/runner/pytest_workflow_*/*/log.err
```
Comment (Emiller88) [inline .github/workflows/pytest-workflow-all.yml:84]: Drop my commits before merging

**Review by Emiller88 — CHANGES_REQUESTED** [review]
Just leaving this as a reminder to drop my commits before someone merges this while I'm asleep.

**Comment by heuermh** [comment]
Closing in favor of #525

**Pull request closed without merging**